front/components/assistant/conversation/AgentSuggestion.tsx

```diff
@@ -0,0 +1,167 @@
+import { Button, DropdownMenu, RobotIcon } from "@dust-tt/sparkle";
+
+import { useAgentConfigurations } from "@app/lib/swr";
+import { AgentConfigurationType } from "@app/types/assistant/agent";
+import {
+  ConversationType,
+  isAgentMention,
+  isUserMessageType,
+  UserMessageType,
+} from "@app/types/assistant/conversation";
+import { WorkspaceType } from "@app/types/user";
+
+export function AgentSuggestion({
+  owner,
+  userMessage,
+  conversation,
+}: {
+  owner: WorkspaceType;
+  userMessage: UserMessageType;
+  conversation: ConversationType;
+}) {
+  const { agentConfigurations } = useAgentConfigurations({
+    workspaceId: owner.sId,
+  });
+  agentConfigurations.sort((a, b) => compareAgentSuggestions(a, b));
+
+  return (
+    <div className="pt-4">
+      <div className="text-xs font-bold text-element-600">
+        Which KillerZorg would you like to talk with?
+      </div>
+      <div className="mt-3 flex items-center gap-2">
```

Comment:
Here you should use <Button.List>

Reply:
Additional question @Duncid => on the figma, between the user's message and the agent suggestion there is a gap of 13 (so 52px) decomposed into a gap-4 + padding top 9

It seemed very spacy, so I did gap-4 + padding top 4 instead (that's what you see on the screenshot)

I can turn it to pt-9 of course, just wanted to check if the large gap was intended?